### DISCUSSION TITLE
Blogging in Djot instead of Markdown

### DISCUSSION THREAD

**Discussion by treeman** [jgm/djot#269] [Show and tell]
I wrote a blog post on how I've started to use Djot instead of CommonMark for my blog.

I migrated from pulldown-cmark to Jotdown, and I have a few extensions to the markup I discuss.

Overall, I'm a fan.

**Comment by matklad** [comment]
@treeman I think your forgot this:

https://www.jonashietala.se/blog/2024/02/02/blogging_in_djot_instead_of_markdown/

:)

**Reply by treeman** [reply]
Yeah... Always nice when you forget the most important part.

**Comment by dereuromark** [comment]
Yeah I started the same for my wordpress blog, rolling it out as we speak: https://github.com/php-collective/wp-djot
